share/completions/sv.fish

```diff
@@ -0,0 +1,80 @@
+# Completions for sv
+# A part of the runit init system
+# Author: Leonardo da Rosa Eugênio <lelgenio@disroot.org>
+
+
+set -l commands \
+    status up down once s u d o pause cont hup \
+    alarm interrupt quit 1 2 term kill exit p c h \
+    a i q 1 2 t k e start stop reload restart \
+    shutdown force-stop force-reload force-restart force-shutdown \
+    try-restart check
+
+
+function __fish_complete_sv_list_services
+    for dir in "$SVDIR" /run/runit/service/ /etc/runit/current /etc/runit/runsvdir/
+        test -d "$dir"
+        and break
+    end
+    set -l services (command ls "$dir")
+    if sv status "$services[1]" &> /dev/null
+        for service in $services
+            printf "%s\t%s\n" "$service" (sv status "$service")
+        end
+    else
+        printf "%s\n" $services
```

Comment:
when does this happen?

Reply:
If the user does not have read access to the services dir.
You can't know the status of a service, but you know what services exist.
```sh
$ sv status elogind
warning: elogind: unable to open supervise/ok: access denied
```